packages/app-root/src/app/groups/[id]/page.js

```diff
@@ -0,0 +1,13 @@
+'use client'
+
+import { GroupStats } from '@zooniverse/user'
+import auth from 'panoptes-client/lib/auth'
+
+export default function GroupPage({ params }) {
+  return (
+    <GroupStats
+      authClient={auth}
+      groupID={params.id}
```

Comment:
Is `id` ok or should it be `group_id`?

Reply:
I think `groupID` is better as `params.id` doesn't tell us what kind of object we're working with. Once we get inside of GroupStats its abundantly clear when its `groupID`. I'd probably change it to `groupId` though to be more consistent in our usage of camelcase.